pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -306,7 +287,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				// pull images
 				// authorizationapi.NewRule("get").Groups(imageGroup).Resources("imagestreams/layers").RuleOrDie(),
 
-				authorizationapi.NewRule(read...).Groups(projectGroup).Resources("projects").RuleOrDie(),
+				authorizationapi.NewRule("get").Groups(projectGroup).Resources("projects").RuleOrDie(),
```

Comment:
ah, granted through the basicuser role, doesn't make sense on a namespace-bound role